cms/djangoapps/contentstore/config/waffle.py

```diff
@@ -2,6 +2,7 @@
 This module contains various configuration settings via
 waffle switches for the contentstore app.
 """
+from openedx.core.djangoapps.waffle_utils import CourseWaffleFlag, WaffleFlagNamespace, WaffleSwitchNamespace
```

Comment:
You can add `CourseWaffleFlag` to the line below instead of duplicating here.

Reply:
Done